Userland/Libraries/LibGUI/MessageBox.cpp

```diff
@@ -103,7 +106,7 @@ void MessageBox::build()
     }
 
     auto& label = message_container.add<Label>(m_text);
-    label.set_fixed_height(16);
```

Comment:
maybe remove this line plus pass `height()` to `set_rect`?

Reply:
That would get rid of the padding, I think the default `height()` is 10px.

It's possible to remove this line and use `total_text_height` in `set_rect`. I think it makes more sense if we keep using `set_fixed_height` and use `max_height()` in `set_rect` because `height()` doesn't update, so I've implemented this now.